pkg/api/v1beta3/types.go

```diff
@@ -971,6 +971,16 @@ type NodeSystemInfo struct {
 	SystemUUID string `json:"systemUUID"`
 	// BootID is the boot-id reported by the node
 	BootID string `json:"bootID" description:"boot id is the boot-id reported by the node"`
+	// Kernel version reported by the node
+	KernelVersion string `json:"kernelVersion" description:"Kernel version reported by the node"`
```

Comment:
Could we say more precisely what this will report (uname -v?) and/or an example?